CHANGES.md

```diff
@@ -58,6 +58,9 @@
   than requiring them to be passed separately via the `--extra_package` option.
   ([#23684](https://github.com/apache/beam/pull/23684))
 
+## Breaking Changes
+
+* `ParquetIO.withSplit` was removed ([#23832](https://github.com/apache/beam/issues/23832)).
```

Comment:
nit: a casual observer might read this as "it's no longer possible to split". Could we add something like "Note: splittable reading has been the default behavior since 2.35.0, the effect of this change is to drop support for _non-splittable_ reading"?

Reply:
Thanks, makes sense, updated (#23866)